src/getdependencies.ts

```diff
@@ -1,24 +1,44 @@
 import { error } from 'console';
+const readYaml =  require('read-yaml');
 
-let readJson;
+let readJson, parseYaml;
 let dependencies = [];
 
-export function getDependencies(repo: string, packageScope: string[]) {
+export function getDependencies(repo: string, packageScope: string[], repoMap: Map<string,string>) {
   try {
-    let path = '../repos/' + repo + '/package.json';
-
+    let path = '../' + repoMap.get(repo) + '/' + repo + '/package.json';
     dependencies = [];
     readJson = require(path).dependencies;
     for (const j in readJson) {
```

Comment:
This should be named better, `i`, `j`, and the such are usually used for index arrays, E.G. `for (let i = 0; i < 10; i++) {`. Since what this is reading is the `package.json`'s dependencies you should be able to name this `dependencies`.